zebra/zebra_rnh.c

```diff
@@ -541,7 +541,7 @@ static int check_overlay_nexthop(struct prefix *pp, uint8_t *isreachable)
 	*isreachable = 1;
 
 	inet_ntop(g_infovlay_prefix.family, &g_infovlay_prefix.u.prefix, via, PREFIX2STR_BUFFER);
-	zlog_debug("Overlay Prefix %s", via); 
+	//zlog_debug("Overlay Prefix %s", via);
```

Comment:
@ganeshsfff can you make it conditional on `IS_ZEBRA_DEBUG_NHT` pls?

Reply:
@sunilfff  Done. Please review once again. 

notify @ganeshkfff